docs/usage/private-modules.md

```diff
@@ -102,7 +102,7 @@ It can be confusing for people who host their own source code privately to be as
 
 Currently the preferred way to configure `github.com` credentials for self-hosted Renovate is:
 
-- Create a read-only Personal Access Token (PAT) for a `github.com` account. It can be any account, and may even be best to be an empty account created just for this purpose.
+- Create a read-only Personal Access Token (PAT) for a `github.com` account. This can be any GitHub account, it might be better to create a "empty" account just for this purpose.
```

Comment:
The general feeling that I get from this document is that we have some "you might like", "it might be good to", "it may even be best" type of sentences. I don't think this helps the document read clearly.

If we have strong opinions on the best way to configure Renovate bot, we should say so clearly in the docs: "This is the best way, don't do it another way". The current wording is a bit too vague for my taste.

As an example, if we think it's better to create a empty/shell GitHub account for the sole purpose of creating the Personal Access Token (PAT), then we should say so clearly.